VR Maze/Assets/Scripts/IMazeGenerator.cs

```diff
@@ -10,6 +10,6 @@ namespace Assets.Scripts
     /// </summary>
     interface IMazeGenerator
     {
-        Maze generate(int width, int height);
+       // Maze generate(int width, int height);

```

Comment:
Does this need to be commented out? I'm not seeing anything here that implements this interface, so keeping this in shouldnt break anything. It's just something that's really easy to forget to uncomment later, as it's such a small piece of code